docs/powervs-faq.md

```diff
@@ -0,0 +1,24 @@
+# PowerVS Frequently Asked Questions
+
+---
+## When Building out a Cluster
+
+#### VM Console shows "grub>"
+Not the end of the world. Just ESC and ENTER, as the message on screen instructs.
+
+#### RHCOS Ignition stalls with "version mismatch"
+The version of RHCOS you are booting up with on cluster nodes cannot handle what the OCP build/version is serving as config for cluster nodes. Make sure they match at least the major and minor versions (eg. 4.5) in your `var.tfvars` file.
+
+---
+## Problems on a Running Cluster
+
+#### Need to add more CPU/Memory to Node
+Go to <https://cloud.ibm.com>, then to your PowerVS service instance, find your VM that backs the cluster Node (eg. myocp-worker-0), click to open up the details page, then "Edit details," update the CPU/Memory with new values, and Save. You should get "Edit successful" message, and your OCP cluster should recognize the change and start showing in Compute section of Web Console. If something fails along the way, try "OS Shutdown" action, edit CPU/Memory as necessary while in "Shutoff" state, and restart the VM.
```

Comment:
Post addition of CPU and memory to node, the node will require restart for the updated resources to get reflected in OCP scheduler. We should mention this explicitly

Reply:
I thought so, but I did work with Support and saw it reflected *without restarting the VM*. That's the basis of the above writeup, with "if something fails along the way..."

Reply:
The VM will have the resources, but it won't be available to OCP. kubelet/crio etc are not equipped to handle dynamic CPU/mem addition yet.

Reply:
Let me challenge you one last time, @bpradipt, as I **saw** it working without bouncing the VM. I admit the "lucky case" was 1 out of 6 that I had to resize, but I swear it **did happen** just a few days ago. And I have that "*if something fails along the way*" clause to cover the reboot process that may be needed (in most cases). Are you still not happy with the proposed answer?

Reply:
@miyamotoh I stand corrected. I did see VM changes getting reflected. Give me sometime. I need to do few more investigations and then we can finalize this.

Reply:
Glad to hear somebody else also sees it. Will wait... ☕